Cargo.toml

```diff
@@ -495,6 +495,7 @@ generic-array = "0.14"
 humantime = "2.1"
 humantime-serde = "1.1"
 itertools = "0.13"
+libloading = "0.8.5"
```

Comment:
this dependency is unused now, can be removed

Reply:
yes, I just trying to test the macro
found that it was a mess with pointers on loading. Just want to represent load function and simple test for load minimal exex

Reply:
@shekhirin can I commit with test and changes on macro? So, that is key changes, cause we need to have correct and worked macro on this PR I think